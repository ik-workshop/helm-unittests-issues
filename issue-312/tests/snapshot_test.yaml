---
templates:
  - templates/deploy.yaml
  - templates/service.yaml
tests:
  - it: should create find document
    documentSelector:
      path: kind
      value: Deployment
    asserts:
      - matchSnapshot: {}
