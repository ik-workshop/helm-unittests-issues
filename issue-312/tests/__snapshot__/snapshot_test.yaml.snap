should create find document:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      name: name
  2: |
    apiVersion: v1
    kind: Service
    metadata:
      name: example-service
